shell/app/common/components/table/index.tsx

```diff
@@ -89,6 +89,8 @@ function WrappedTable<T extends object = any>({
     : (paginationProps as TablePaginationConfig);
   const { current = 1, pageSize = PAGINATION.pageSize } = pagination;
 
+  const containerRef = React.useRef(document.body);
```

Comment:
how about define as `React.useRef< HTMLElement >(document.body);` then maybe dont need `as ` in line 209